packages/web-components/src/menu/menu.template.ts

```diff
@@ -0,0 +1,15 @@
+import { elements, ElementViewTemplate, html, ref, slotted } from '@microsoft/fast-element';
+import type { Menu } from './menu.js';
+
+export function menuTemplate<T extends Menu>(): ElementViewTemplate<T> {
+  return html<T>`
+    <template @keydown="${(x, c) => x.keydownHandler(c.event as KeyboardEvent)}">
+      <slot name="trigger" ${slotted({ property: 'trigger', filter: elements() })} tabindex="0"></slot>
```

Comment:
Slots should not have a tabindex, they aren't included in the a11y tree - is there a reason this is needed?